### PR TITLE
FPL-46: Implemented orders and hearing tab

### PR DESCRIPTION
**Pull request by Jamesnelson117**
**Before creating a pull request make sure that:**

- [x] commit messages are meaningful and follow good commit message guidelines
- [x] README and other documentation has been updated / added (if needed)
- [x] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/FPL-46

### Change description ###
Added orders and hearing tab

screen shot 2018-10-24 at 17 15 28

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x ] No
```

### CODE REVIEW

File: e2e/paths/enterOrdersAndDirections_test.js [hunk 1]
```diff
@@ -7,19 +7,22 @@ Before((I, caseViewPage) => {
   caseViewPage.goToNewActions(config.applicationActions.selectOrders);
 });
 
-Scenario('Select the care order case order and continue', (I, ordersNeededPage) => {
+Scenario('Select the care order case order and continue', (I, caseViewPage, ordersNeededPage) => {
   ordersNeededPage.checkCareOrder();
   I.continueAndSubmit(config.eventSummary, config.eventDescription);
-  I.seeEventSubmissionConfirmation(config.applicationActions.selectOrders);
```
Comment (geek-soft) [inline e2e/paths/enterOrdersAndDirections_test.js]: I guess we would like to see this back.

File: e2e/paths/enterOrdersAndDirections_test.js [hunk 2]
```diff
@@ -7,19 +7,22 @@ Before((I, caseViewPage) => {
...
-Scenario('Select all case orders and fill in directions & interim information', (I, ordersNeededPage) => {
+Scenario('Select all case orders and fill in directions & interim information', (I, caseViewPage, ordersNeededPage) => {
   ordersNeededPage.checkCareOrder();
   ordersNeededPage.checkSupervisionOrder();
   ordersNeededPage.checkEducationSupervisionOrder();
   ordersNeededPage.checkEmergencyProtectionOrder();
   ordersNeededPage.checkOtherOrder();
   ordersNeededPage.enterDirectionAndInterim();
   I.continueAndSubmit(config.eventSummary, config.eventDescription);
-  I.seeEventSubmissionConfirmation(config.applicationActions.selectOrders);
```
Comment (geek-soft) [inline e2e/paths/enterOrdersAndDirections_test.js]: Same here

File: e2e/pages/ordersNeeded/ordersNeeded.js [hunk 3]
```diff
@@ -3,15 +3,15 @@ const I = actor();
 module.exports = {
   fields: {
     orderType: {
-      careOrder: 'Care order',
-      supervisionOrder:'Supervision order',
-      educationSupervisionOrder: 'Education supervision order',
-      emergencyProtectionOrder: 'Emergency protection order',
-      other: 'Other',
+      careOrder: {css: 'input[id="orders_orderType-Care order"]'},
```
Comment (kurt-uk) [inline e2e/pages/ordersNeeded/ordersNeeded.js]: this isnt selecting by css? youre using the id, so why not have the short hand id?

Comment (geek-soft) [inline e2e/pages/ordersNeeded/ordersNeeded.js]: Do we need to make that change?

File: e2e/pages/ordersNeeded/ordersNeeded.js [hunk 4]
```diff
@@ -3,15 +3,15 @@ const I = actor();
...
+      supervisionOrder: {css: 'input[id="orders_orderType-Supervision order"]'},
+      educationSupervisionOrder: {css: 'input[id="orders_orderType-Education supervision order"]'},
+      emergencyProtectionOrder: {css: 'input[id="orders_orderType-Emergency protection order"]'},
+      other: '#orders_orderType-Other',
```
Comment (kurt-uk) [inline e2e/pages/ordersNeeded/ordersNeeded.js]: down to here

File: e2e/paths/selectHearing_test.js [hunk 5]
```diff
@@ -7,19 +7,26 @@ Before((I, caseViewPage) => {
...
+Scenario('completing the Select hearing section of the c110a application', (I, caseViewPage, selectHearingPage) => {
   selectHearingPage.enterTimeFrame();
   selectHearingPage.enterHearingType();
   selectHearingPage.enterWithoutNoticeHearing();
   selectHearingPage.enterReducedHearing();
   selectHearingPage.enterRespondentsAware();
   I.continueAndSubmit(config.eventSummary, config.eventDescription);
-  I.seeEventSubmissionConfirmation(config.applicationActions.selectHearing);
+  caseViewPage.selectTab(caseViewPage.tabs.ordersHearing);
+  I.seeAnswerInTab('Hearing', 'Give reason', 'test reason');
```
Comment (geek-soft) [inline e2e/paths/selectHearing_test.js]: Assertion for `When do you need a hearing?` is missing.

File: e2e/pages/ordersNeeded/ordersNeeded.js [hunk 6]
```diff
@@ -3,11 +3,11 @@ const I = actor();
 module.exports = {
   fields: {
     orderType: {
-      careOrder: {css: 'input[id="orders_orderType-Care order"]'},
-      supervisionOrder: {css: 'input[id="orders_orderType-Supervision order"]'},
-      educationSupervisionOrder: {css: 'input[id="orders_orderType-Education supervision order"]'},
-      emergencyProtectionOrder: {css: 'input[id="orders_orderType-Emergency protection order"]'},
-      other: '#orders_orderType-Other',
+      careOrder: 'Care order',
```
Comment (kurt-uk) [inline e2e/pages/ordersNeeded/ordersNeeded.js]: so this is selecting with the text, where we can, were using id's

**Pull request merged**